airflow/settings.py

```diff
@@ -503,6 +503,15 @@ def import_local_settings():
             setattr(airflow_local_settings, "task_policy", airflow_local_settings.policy)
             names.remove("policy")
 
+        if "SMTP_DEFAULT_TEMPLATED_SUBJECT" in names or "SMTP_DEFAULT_TEMPLATED_HTML_CONTENT_PATH" in names:
+            warnings.warn(
+                "Configuring non-default `SMTP_DEFAULT_TEMPLATED_SUBJECT` and "
+                "`SMTP_DEFAULT_TEMPLATED_HTML_CONTENT_PATH` is deprecated. Please upgrade to "
+                "the new version of the SMTP provider and use provider configurations instead.",
+                AirflowProviderDeprecationWarning,
+                stacklevel=2,
+            )
+
```

Comment:
I'm not sure do we need this check if we consider that this settings originally added by a mistake.
Let's wait what the other say.

Reply:
Yes. We should IMHO yank 1.6.0 and forget about those old settings.